src/pages/Main.tsx

```diff
@@ -1,79 +1,12 @@
-import { useState } from "react";
-import {
-  Context,
-  H1,
-  HorizontalDivider,
-  LoadingSpinner,
-  Property,
-  proxyFetch,
-  Stack,
-  useDeskproAppEvents,
-  useInitialisedDeskproAppClient,
-} from "@deskpro/app-sdk";
-import * as React from "react";
-/*
-    Note: the following page component contains example code, please remove the contents of this component before you
-    develop your app. For more information, please refer to our apps
-    guides @see https://support.deskpro.com/en-US/guides/developers/anatomy-of-an-app
-*/
+import { useEffect } from "react";
+import { useNavigate } from "react-router-dom";
 
 export const Main = () => {
-  const [ticketContext, setTicketContext] = useState<Context | null>(null);
-  const [examplePosts, setExamplePosts] = useState<
-    { id: string; title: string }[]
-  >([]);
+  const navigate = useNavigate();
 
-  // Add a "refresh" button @see https://support.deskpro.com/en-US/guides/developers/app-elements
-  useInitialisedDeskproAppClient((client) => {
-    client.registerElement("myRefreshButton", { type: "refresh_button" });
+  useEffect(() => {
+    navigate("/findCreate/account");
```

Comment:
I think, firstly we need to try automatically linking the contact first. We do it at HubSpot App https://github.com/DeskproApps/hubspot/blob/master/src/pages/Main.tsx#L20

1. get email in the deskpro context
2. try to find a contact by email in the Xero
3.1. found - automatically linked and navigate to the home page
3.2 didn't find - navigate to the "search and linking" page

Reply:
I've already done it, it's going on another pr

Reply:
already done, but belongs to another pr